builtin/logical/mysql/backend_test.go

```diff
@@ -220,14 +248,26 @@ func testAccStepConfig(t *testing.T, d map[string]interface{}, expectError bool)
 	}
 }
 
-func testAccStepRole(t *testing.T) logicaltest.TestStep {
+func testAccStepRole(t *testing.T, wildCard bool) logicaltest.TestStep {
+
+	if wildCard == true {
```

Comment:
Rather than duplicate all of this, please assign the value to be returned to a variable, then simply add the extra value (or not) based on the passed-in bool.